app/services/irs_attempts_api/tracker.rb

```diff
@@ -86,9 +80,5 @@ def ignore_idv_event?(event_type)
       !IdentityConfig.store.irs_attempt_api_idv_events_enabled &&
```

Comment:
I think we can remove this `ignore_idv_event?` logic as well. this class can pretty much be empty?